management/tenant.go

```diff
@@ -198,6 +198,42 @@ type TenantFlags struct {
 	// If enabled, this will use the scope description when generating a consent
 	// prompt. Otherwise, the scope name is used.
 	UseScopeDescriptionsForConsent *bool `json:"use_scope_descriptions_for_consent,omitempty"`
+
+	// Whether the legacy delegation endpoint will be enabled for your account (true) or not available (false).
+	AllowLegacyDelegationGrantTypes *bool `json:"allow_legacy_delegation_grant_types,omitempty"`
```

Comment:
Let's not add any legacy enabling flags or dashboard flags, it's extremely unlikely they'll ever be used. Let's just keep:

- EnableIDTokenAPI2
- NoDisclosureEnterpriseConnections
- DisableManagementAPISMSObfuscation
- EnableADFSWAADEmailVerification
- RevokeRefreshTokenGrant
- DisableFieldsMapFix

Reply:
Update: Discussed this internally, we're gonna stay consistent with the [auth0](https://auth0.com/docs/api/management/v2#!/Tenants/patch_settings) docs.